build.sbt

```diff
@@ -20,6 +20,8 @@ ThisBuild / organization := "za.co.absa.login-service"
 
 lazy val scala212 = "2.12.17"
 
+ThisBuild / name := "login-service"
```

Comment:
This shouldn't be there

Reply:
Hi Sasa

Thank you for the review. I've made changes as per the comments. It should look better now?